assets/blocks/quiz/quiz-block/quiz-edit.js

```diff
@@ -22,7 +22,11 @@ const QuizEdit = ( props ) => {
 			<div className="sensei-lms-quiz-block__separator">
 				<span>{ __( 'Lesson Quiz', 'sensei-lms' ) }</span>
 			</div>
-			<InnerBlocks allowedBlocks={ [ 'sensei-lms/quiz-question' ] } />
+			<InnerBlocks
+				allowedBlocks={ [ 'sensei-lms/quiz-question' ] }
+				template={ [ [ 'sensei-lms/quiz-question', {} ] ] }
+				templateInsertUpdatesSelection={ false }
```

Comment:
Note that this only works well when the quiz block is added as a lesson template. 
If the user removes the quiz block and adds it later, it will
- not focus the first question block
- `useAutoInserter` will add a second question block, as it doesn't pick up the one added by the template. It will also focus this second block with the current `selectFirstBlock: true` parameter.

I think it could be a common enough use case to handle it, and make the block work well when inserted later.

We could introduce an `isPostTemplate` attribute and make this behave differently depending on that, changing the lesson post template part to:

```php
[ 'sensei-lms/quiz', [ 'isPostTemplate' => true ] ]
```

And InnerBlocks config to:
```jsx
<InnerBlocks
	allowedBlocks={ [ 'sensei-lms/quiz-question' ] }
	template={
		isPostTemplate ? [ [ 'sensei-lms/quiz-question', {} ] ] : []
	}
	templateInsertUpdatesSelection={ false }
/>
```

Reply:
Good catch! And thank you for the fix suggestion! I used that. ;)